libs/paypal/src/drivers/magento/paypal.service.ts

```diff
@@ -16,15 +16,15 @@ import { DaffPaypalConfig } from '../injection-tokens/paypal-config.token';
 @Injectable({
   providedIn: 'root'
 })
-export class DaffMagentoPaypalService<T extends DaffPaypalTokenRequest, V extends DaffPaypalTokenResponse> implements DaffPaypalServiceInterface<T, V> {
+export class DaffMagentoPaypalService implements DaffPaypalServiceInterface {
   
   constructor(
     private apollo: Apollo,
-		@Inject(DaffPaypalTransformer) private transformer: DaffPaypalTransformerInterface<T, V>,
+		@Inject(DaffPaypalTransformer) private transformer: DaffPaypalTransformerInterface<DaffPaypalTokenRequest, DaffPaypalTokenResponse>,
```

Comment:
The transformer interface and injection token can be removed. That can be done in a separate PR though.

Reply:
yea, will do in another PR